packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Consumer } from 'xfc';
+import './EmbeddedContentConsumer.scss';
+
+const propTypes = {
+  /**
+   * The source URL of the content to load.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * An object containing event handlers keyed by the event name.
+   * (e.g. {'eventA': function() {}, 'eventB': function() {}})
+   */
+  eventHandlers: PropTypes.object,
+};
+
+const defaultProps = {
+  src: undefined,
+  eventHandlers: {},
+};
+
+class EmbeddedContentConsumer extends React.Component {
+
+  componentDidMount() {
+    if (this.embeddedContentWrapper) {
```

Comment:
I think I had this in the prototype, but the if check here isn't necessary. If the div didn't mount before componentDidMount is called, we'd be having bigger problems.